docs/src/getting-started/installation.md

```diff
@@ -329,6 +330,76 @@ Make sure to include this section in `Scarb.toml`
 test = "snforge test"
 ```
 
+## Set up shell completions (optional)
+
+<details>
+  <summary><strong>Bash</strong></summary>
+
+Completions are configured by doing the following:
+```shell
+mkdir -p "${ASDF_DATA_DIR:-$HOME/.asdf}/completions"
+sncast completion bash > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/sncast.bash"
+snforge completion bash > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/snforge.bash"
+```
+
+Then add the following to your `.bash`:
+```shell
+# source completion scripts
+. "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/sncast.bash"     
+. "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/snforge.bash"
+```
+
+</details>
+
+<details>
+  <summary><strong>ZSH</strong></summary>
+
+Completions are configured by doing the following:
+```shell
+mkdir -p "${ASDF_DATA_DIR:-$HOME/.asdf}/completions"
+sncast completion zsh > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/_sncast"
+snforge completion zsh > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/_snforge"
+```
+
+Then add the following to your `.zshrc`:
+```shell
+# append completions to fpath
+fpath=(${ASDF_DATA_DIR:-$HOME/.asdf}/completions $fpath)
+# initialise completions with ZSH's compinit
+autoload -Uz compinit && compinit
+```
+This is to enable autocompletion in [ZSH](https://wiki.archlinux.org/title/Zsh#Command_completion).
+
+</details>
+
+<details>
+  <summary><strong>Fish</strong></summary>
+
+Completions are configured by doing the following:
+```shell
+sncast completion fish > ~/.config/fish/completions/sncast.fish
+snforge completion fish > ~/.config/fish/completions/snforge.fish
+```
+
+</details>
+
+<details>
+  <summary><strong>Elvish</strong></summary>
+
+Completions are configured by doing the following:
+
+```shell
+sncast completion elvish >> ~/.config/elvish/rc.elv
+snforge completion elvish >> ~/.config/elvish/rc.elv
+```
+
+</details>
+
+<details>
+  <summary><strong>PowerShell</strong></summary>
+    📄 Doc is under construction.
+</details>
```

Comment:
Don't have a strong opinion here, but I think we can drop it as there is no guide for powershell yet.

Alternatively if we want to keep it (in order to show that it's in progress) then maybe let's use 🏗️ emoji 😅 .